app/static/js/app.js

```diff
@@ -293,6 +293,21 @@ function setCursor(cursor, save = true) {
   }
 }
 
+function setFullScreen() {
+  let el = document.getElementById("remote-screen");
```

Comment:
Can `el` be `const remoteScreen`? I find it easier to reason about the variables when they have descriptive names. And we can `const` here.